bridge/slack/slack.go

```diff
@@ -302,6 +307,39 @@ func (b *Bslack) sendRTM(msg config.Message) (string, error) {
 	return "slack " + id, nil
 }
 
+// handleUploadFile handles native upload of files
```

Comment:
This file was copy-pasted without changes from the `handlers.go` file as it is actually not a handler and part of the outgoing pipeline, not the incoming one.